tools/check_format_compatible.sh

```diff
@@ -76,7 +76,7 @@ https_proxy="fwdproxy:8080" git fetch github_origin
 for checkout_obj in "${checkout_objs[@]}"
 do
    echo == Generating DB from "$checkout_obj" ...
-   git checkout origin/$checkout_obj
```

Comment:
The reason I added it was that in my local checkout, without origin it doesn't work anymore. A new repo clone works though. I thought adding a origin would just solve the problem. Looks like it doesn't.